web/lib/api/call-endpoint.ts

```diff
@@ -0,0 +1,21 @@
+import { auth } from 'web/lib/firebase/users'
+import { PlaceBetParams } from 'web/pages/api/v0/bet'
+
+export function placeBet(params: PlaceBetParams) {
+  return call('/api/v0/bet', params)
+}
```

Comment:
I guess no matter what direction we go with running these API endpoints, we could put some types for them in `common` and use them like this.

Reply:
Yeah, I aways thought it was bad that we didn't type our api calls.